brangelina.vim

```diff
@@ -138,10 +135,9 @@ nnoremap <leader>gg :Twiggy<cr>
 augroup customCommands
   autocmd!
   autocmd FileType markdown nnoremap <localleader>m :LivedownToggle<cr>
-  autocmd FileType javascript nnoremap <localleader>c :JSContextColorToggle<cr>
+  autocmd FileType javascript nnoremap <localleader>co :JSContextColorToggle<cr>
```

Comment:
Also happy to remove this plugin. I'm not using it anymore.

Reply:
I think there is more stuff that we could remove, wanna go over it some time?

Reply:
Sounds good!